examples/di/InfiAgent-DABench/run_InfiAgent-DABench.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+import json
+
+from DABench import DABench
+
+from metagpt.roles.di.data_interpreter import DataInterpreter
+
+
+async def get_prediction(agent, requirement):
+    """Helper function to obtain a prediction from a new instance of the agent.
+
+    This function runs the agent with the provided requirement and extracts the prediction
+    from the result. If an error occurs during processing, it logs the error and returns None.
+
+    Args:
+        agent: The agent instance used to generate predictions.
+        requirement: The input requirement for which the prediction is to be made.
+
+    Returns:
+        The predicted result if successful, otherwise None.
+    """
+    try:
+        # Run the agent with the given requirement and await the result
+        result = await agent.run(requirement)
+
+        # Parse the result to extract the prediction from the JSON response
+        prediction_json = json.loads(str(result).split("Current Plan")[1].split("## Current Task")[0])
+        prediction = prediction_json[-1]["result"]  # Extract the last result from the parsed JSON
+
+        return prediction  # Return the extracted prediction
+    except Exception as e:
+        # Log an error message if an exception occurs during processing
+        print(f"Error processing requirement: {requirement}. Error: {e}")
+        return None  # Return None in case of an error
+
+
+async def evaluate_all(agent, k):
+    """Evaluate all tasks in DABench using the specified baseline agent.
+
+    Tasks are divided into groups of size k and processed in parallel.
+
+    Args:
+        agent: The baseline agent used for making predictions.
+        k (int): The number of tasks to process in each group concurrently.
+    """
+    DA = DABench()  # Create an instance of DABench to access its methods and data
```

Comment:
I prefer to use **bench** as a variable. Generally speaking, python PEP8 does not allow uppercase variable names

Reply:
done